### PR TITLE
Release 2.0.2 of Streams and HTTP

### PR DESCRIPTION
**Pull request by ktoso**

### CODE REVIEW

**Comment by ktoso** [comment]
cc @drewhk @2m @rkuhn @johanandren

**Comment by johanandren** [comment]
LGTM

**Comment by johanandren** [comment]
Also, great work!

**Comment by ktoso** [comment]
Thanks, added links to issues.

Checked the release as well from staging so I'll just click the buttons ;)

**Comment by drewhk** [comment]
LGTM

**Comment by ktoso** [comment]
Thanks :)

File: news/_posts/2015-01-14-akka-streams-2.0.2-released.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+- scalac bug when using members of MediaTypes in a map [#19418](https://github.com/akka/akka/issues/19418)
+
+The reference documentation for the 2.0.2 release can be found here for [Java](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0.2/java.html) and [Scala](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0.2/scala.html).
+
+If you are upgrading from the old 1.0.x series, you may want to read the following 
+[migration guide for Java](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0.2/java/migration-guide-1.0-2.x-java.html) 
+or [migration guide for Scala](http://doc.akka.io/docs/akka-stream-and-http-experimental/2.0.2/scala/migration-guide-1.0-2.x-scala.html).
+
+The full list of changes since the last milestone is available under the [streams-2.0.2](https://github.com/akka/akka/issues?q=is%3Aissue+milestone%3Astreams-2.0.2+is%3Aclosed) 
+and [http-2.0.2](https://github.com/akka/akka/issues?q=is%3Aissue+milestone%3Ahttp-2.0.2+is%3Aclosed) milestones on github for your reference.
```
Comment (rkuhn) [inline news/_posts/2015-01-14-akka-streams-2.0.2-released.md]: the links are wrong here as well (it is all on stream-http-2.0.2)

Comment (ktoso) [inline news/_posts/2015-01-14-akka-streams-2.0.2-released.md]: I'll fix here

**Pull request merged**